script/libtest.sh

```diff
@@ -58,12 +58,11 @@ k8s_wait_for_job_completed() {
     while [ "$retryTimeSeconds" -gt 0 ]; do
         # Avoid to exit the function if the job is not completed yet
         set +e
-        kubectl get jobs -n $namespace -l $labelSelector \
-           -o jsonpath='{.items[*].status.conditions[?(@.type=="Complete")].status}' | grep "True"
+        kubectl get jobs -n $namespace -l $labelSelector -o jsonpath='{.items[*].status.conditions[?(@.type=="Complete")].status}' | grep "True"
```

Comment:
Removed the line split since I found that adding a trailing space `\ ` make it fail. It's not the cause of the error here but it's safer to have it in just one line.

Reply:
Is it? Why? I mean we use line split everywhere. What error were you seeing?

Reply:
I mean that while I was copying this code I accidentally added a trailing space after the `\`. The code in the repo was fine but while I was debugging I found that somehow I introduced the trailing space. To avoid some issue like that in the future I removed the splitting of the command. It's not that important though, we can keep using `\` if the command is too long.